code/game/turfs/turf.dm

```diff
@@ -179,14 +178,17 @@
 
 	lighting_overlay = old_lighting_overlay
 
+	recalc_atom_opacity()
+	lighting_overlay = old_lighting_overlay
```

Comment:
You need to put everything lighting related (except atom opacity recalculation) behind an if for whether the lighting corners initialized (see my comment on the process about it).

Why? If you don't the lighting code will royally break if a `changeTurf` happens before the process setup runs.